### PR TITLE
IIS_2617: Migrating tile_map to map in dashboards

### PR DESCRIPTION
**Pull request by ishleenk17**
- Enhancement

## What does this PR do?

This PR replaces the tile_map object with map in IIS Logs dashboard

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## How to test this PR locally

Load the newly created dashboard and we should see map as the object 

- Closes https://github.com/elastic/integrations/issues/2617

## Screenshots
Below screenshot shows map object updated in IIS

Screenshot 2022-05-26 at 1 46 12 PM

### CODE REVIEW

**Review by andrewkroh — COMMENTED** [review]
The conversion changes LGTM.

I think this would be a good opportunity to "unlink" the visualizations such that they are embedded into the dashboard.

File: packages/iis/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "0.8.6"
+  changes:
+    - description: Migrating the tile_map to map object in dashboard
+      type: enhancement
```
[the remark this answers is not in the document]
Comment (ishleenk17) [reply packages/iis/changelog.yml:5]: @andrewkroh : We have a different task where we will be taking IIS to GA.
So I would like to bump the major in that. 
In that case I'll change it to 0.9.0 ?

**Comment by ishleenk17** [comment]
@andrewkroh @flash1293 : I am doing this migration using the latest stack 8.2. 
Hence we see the visualisations/dashboards/maps in 8.2.  As a result of this we have to drop all the previous kibana versions supported (7.14 an 8.0). 

As per the issue the editor for tile map is removed in 8.0 and map was added. 
Shall we do the change with older 8.0 ELK stack ? In that case we would have to drop support for only 7.14. 
Please share your inputs here ?

cc: @lalit-satapathy

**Review by andrewkroh — APPROVED** [review]
Assuming the "View in Maps" conversion action only exists in 8.x then I am good with these changes.

**Comment by ishleenk17** [comment]
> > I am doing this migration using the latest stack 8.2.
> > Hence we see the visualisations/dashboards/maps in 8.2. As a result of this we have to drop all the previous kibana versions supported (7.14 an 8.0).
> 
> I agree with @jsoriano that it would be best to use a 7.17 so that we can maintain compatibility for users still on 7.x, but AFAIK the "View in Maps" button that converts the tile_map for you only exists in 8.0+. You can check if it exists in 7.17, and if not then it will need to drop the 7.x compatibility or find a different way to do the conversion.

Yes, just checked. The view maps works for map objects from 8.0 stack. So, I am using that stack for the dashboards.
This will result in dropping support for 7.14, but still supporting ^8.0.0

**Review by muthu-mps — APPROVED** [review]
LGTM!

**Pull request merged**